synfig-studio/src/gui/dialogs/dialog_guide.cpp

```diff
@@ -137,6 +166,12 @@ Dialog_Guide::on_ok_or_apply_pressed(bool ok)
 		curr_guide->angle = synfig::Angle::rad(angle_widget->get_value());
 	}
 
+	if(curr_guide->point[0] != (x_widget->get_value())/60){
+		curr_guide->point[0] =  (x_widget->get_value())/60;
+	}
+	if(curr_guide->point[1] != (y_widget->get_value())/60){
+		curr_guide->point[1] =  (y_widget->get_value())/60;
+	}
```

Comment:
1. why the `if` condition? After those lines you force `curr_guide->point[0]` to be `x_widget->get_value())/60` (and similar to `curr_guide->point[1]`)
2. prefer the style `if (condition) {` (note the spaces I 'wrote')
3. why so many spaces after assignment sign?
4. why the parenthesis surrounding `x_widget->get_value()`?
5. the magic number 60 is true for the default canvas settings. To get the actual value, you have to look to RendDesc value of Canvas.

Reply:
i tried to follow the convention for the if conditions 
<img width="568" alt="image" src="https://github.com/synfig/synfig/assets/82810411/f3ad0b24-bc11-422e-a80d-f5d9b9a72d37">
should i make changes?

Reply:
> should i make changes?

@Afterdie 

I suggest skipping this for now because I also didn't understand that part ("After those lines you force...").
@rodolforg Can you explain it?

Reply:
@ice0 no matter whether the `if` condition is true or false, at the end the variable value will be that one

Reply:
This part is clear, but where is the value setting?
After that code only:

```
	if (ok)
		hide();
	else
		current_work_area->get_drawing_area()->queue_draw();
```

So I don't understand where these values can be set.

Reply:
I think what @rodolforg meant to say was that why perform a check to see if the value has changed why not just set it to `x_widget->get_value()*60` which is valid but for example if the user changes only the angle, theres no need to change the pivot point too if its the same value.

Reply:
Sorry, I meant there is no need for the `if` line. Only its contents directly